### PR TITLE
Refactor to stencil one

### PR DESCRIPTION
**Pull request by manucorporat**

### CODE REVIEW

**Comment by ippeiukai** [comment]
We are struggling with performance tuning chiefly caused by https://github.com/ionic-team/stencil/issues/467.
No pressure, but we're really looking forward to Stencil One update coming to Ionic 4! 🚀 

Let me know if there is anything we can do as a devoted Ionic user.

File: .scripts/release-dev.js [hunk 1]
```diff
@@ -37,7 +38,7 @@ async function main() {
     packages.forEach(package => {
       common.prepareDevPackage(tasks, package, devVersion);
     });
-    common.publishPackages(tasks, packages, devVersion, DIST_TAG);
+    common.publishPackages(tasks, packages, devVersion, DIST_NPM_TAG);
```
Comment (brandyscarney) [inline .scripts/release-dev.js]: Can this be undone now that we will be updating Ionic?

File: angular/package.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
     "css/"
   ],
   "dependencies": {
-    "@ionic/core": "4.5.0",
+    "@ionic/core": "one",
```
Comment (brandyscarney) [inline angular/package.json]: Same with this

File: core/src/components/action-sheet-controller/readme.md [hunk 3]
```diff
@@ -11,12 +11,6 @@ Action Sheet controllers programmatically control the action sheet component. Ac
 
 Create an action sheet overlay with action sheet options.
 
-#### Parameters
```
Comment (brandyscarney) [inline core/src/components/action-sheet-controller/readme.md]: Why was this documentation removed?

File: core/src/components/button/test/basic/index.html [hunk 4]
```diff
@@ -121,7 +121,7 @@
     }
 
     function toggleDisabled() {
-      var buttonEl = this.document.getElementById('disabledButton');
+      var buttonEl = documentument.getElementById('disabledButton');
```
Comment (brandyscarney) [inline core/src/components/button/test/basic/index.html]:
```suggestion
      var buttonEl = document.getElementById('disabledButton');
```

**Pull request merged**